supabase/migrations/20241212204905_online_evaluators.sql

```diff
@@ -0,0 +1,66 @@
+create table "public"."online_evaluators" (
+    "id" bigint generated by default as identity not null,
+    "created_at" timestamp with time zone not null default now(),
+    "evaluator" uuid not null,
+    "config" jsonb,
+    "organization" uuid not null
+);
+
+
+alter table "public"."online_evaluators" enable row level security;
+
+CREATE UNIQUE INDEX online_evaluators_pkey ON public.online_evaluators USING btree (id);
```

Comment:
Let's merge all the migrations into 1 file...

And then let's also REVOKE ALL for anon, authenticated and service_role